tools/cellxgene_census_builder/src/cellxgene_census_builder/build_soma/globals.py

```diff
@@ -338,7 +338,6 @@
     "py.init_buffer_bytes": 1 * 1024**3,
     "py.deduplicate": "true",
     "soma.init_buffer_bytes": 1 * 1024**3,
-    "sm.consolidation.buffer_size": 3 * 1024**3,
```

Comment:
moved to consolidation-specific code to centralize all consolidate/vacuum config.